tensorflow-ops/src/TensorFlow/Ops.hs

```diff
@@ -256,6 +257,15 @@ constant (Shape shape') values
                       [def & TensorShape.size .~ x | x <- shape']
                 & tensorVal .~ values
 
+-- | Reshape a N-D tensor down to a scalar.
+toScalar :: (TensorType a) 
```

Comment:
i've decided i'm torn

ideally one would have a `rank` associated to Tensors, and we could say that `scalarize` goes from `rank n -> rank 0`, whereas `squeeze` would go from `rank n -> rank m` with `m <= n`. and it might be a nice dream to one day have functions that demand `rank 0` tensors.

maybe that's a bit too demanding at this point though.